receiver/jaegerreceiver/factory.go

```diff
@@ -158,14 +153,6 @@ func (f *Factory) CreateTraceReceiver(
 		}
 	}
 
-	if protoTChannel != nil && protoTChannel.IsEnabled() {
```

Comment:
What happens if someone has this in the config file? Would it silently be ignored? Perhaps there could be a warning stating that this has been removed and that gRPC should be used instead?

Reply:
It would currently complain - so fail fast. It is the same issue with the jaeger thrift exporter being removed, and the existing one renamed from `jaeger_grpc` to `jaeger`.

As there is already going to be a breaking change for the next release, my preference would be to just get this one removed as well - so cleaned up before getting to 1.x.